index.js

```diff
@@ -97,6 +97,14 @@ var singleTag = {
   wbr: true,
 };
 
+/*
+  Tags that need to be closed
+*/
+var nonsingleTag = {
+  __proto__: null,
+  iframe: true
```

Comment:
I still don't get why this is a problem in XHTML. Anyway, for just a single case, just replacing this with `=== 'iframe'` is way faster.